infra/data-pipeline.yml

```diff
@@ -234,6 +234,45 @@ tasks:
         owner: release-mgmt-analysis@mozilla.com
         source: ${repository}/raw/master/data-pipeline.yml
 
+    - ID: fixed-comments-retrieval
+      created: { $fromNow: "" }
+      deadline: { $fromNow: "2 days" }
+      expires: { $fromNow: "1 year" }
+      provisionerId: proj-bugbug
+      workerType: compute-small
+      dependencies:
+        - revisions-retrieval
+      payload:
+        env:
+          TC_SECRET_ID: project/bugbug/production
+        maxRunTime: 86400
+        image: mozilla/bugbug-base:${version}
+        command:
+          - "bugbug-data-bugzilla"
```

Comment:
This is supposed to be the comment to run the script: https://github.com/mozilla/bugbug/blob/7b29581e03fa630243bc984e26c89017f23647cb/setup.py#L47

We need to add a command for the new script and use it here.

Also, we should add it to the integration test:

https://github.com/mozilla/bugbug/blob/7b29581e03fa630243bc984e26c89017f23647cb/scripts/integration_test.sh#L16-L19

Reply:
Added the command here: 518f04a9dce3bb5f27923b123527c45d0aa37d3d
Added an integration test here: 76b5db720f69d5950bf81374ea5165cc75f87e2f